articles/machine-learning/how-to-monitor-view-training-logs.md

```diff
@@ -100,7 +100,9 @@ You can view the metrics of a trained model using ```run.get_metrics()```. For e
 
 You can browse completed run records, including logged metrics, in the [Azure Machine Learning studio](https://ml.azure.com).
 
-Navigate to the **Experiments** tab and select your experiment. On the experiment run dashboard, you can see tracked metrics and logs for each run. 
+Navigate to the **Experiments** tab. To view all your runs in your Workspace across Experiments, select the 'All runs' tab. You can drill down on runs for specific Experiments by applying the Experiment filter in the top menu bar. 
+
+For the individual Experiment view, select the 'All experiments' tab. On the experiment run dashboard, you can see tracked metrics and logs for each run. 
```

Comment:
should be **All experiments** tab.

Reply:
the experiments tab is on the left nav so it only reads 'Experiments' as it exists today

Reply:
Oh, so sorry... meant **All runs**.  The point is to use ** instead of ' to highlight a UI element such as a tab!